packages/synthetix-main/contracts/interfaces/IDelegation.sol

```diff
@@ -0,0 +1,15 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IDelegation {
+    enum Permission {
```

Comment:
im assuming we can just add more items to the enum for future upgrades. Is there a check in the deployment tool that verifies that enums are not updated incorrectly?

Reply:
not yet